index.html

```diff
@@ -220,6 +219,95 @@ <h3>Skills</h3>
 			</div>
 		</section>
 
-		<script src="main.js"></script>
-	</body>
-</html>
+        <section id="contact">
+			<div class="pri-text">
+				<p class="contact-para">
+					I'm always interested in hearing about new projects, so if you'd like
+					to chat please get in touch.
+				</p>
+			</div>
+			<form action="https://formspree.io/f/xpznwejk" method="post" class="form">
+				<fieldset>
+					<label class="firstN"
+						><input
+							type="text"
+							name="firstname"
+							id="firstname"
+							maxlength="30"
+							placeholder="First name"
+							required
+						/>
+					</label>
+					<label class="lastN">
+						<input
+							type="text"
+							name="lastname"
+							id="lastname"
+							maxlength="30"
+							placeholder="Last Name"
+							required
+						/>
+					</label>
+					<label class="fullN"
+						><input
+							type="text"
+							name="fullname"
+							id="fullname"
+							maxlength="30"
+							placeholder="Full name"
+							required
+						/>
+					</label>
+
+					<label class="email"
+						><input
+							type="email"
+							name="email"
+							id="email"
+							placeholder="Email address"
+							required
+						/>
+					</label>
+					<label class="msg">
+						<textarea
+							name="msg"
+							id="msg"
+							cols="30"
+							rows="10"
+							maxlength="500"
+							minlength="30"
+							placeholder="Enter text here"
+							required
+						></textarea>
+					</label>
+					<button class="btn" type="submit">Get in Touch</button>
```

Comment:
- [ ] Please make sure that you add the button interactions as mentioned in the figma template. Right now, your hover interaction is a bit different, just one more step to fix it 🏁 I know you can do that 
 - [ ]  Also, your pressed interaction is same as the `enabled` button interaction. You can use `button:active` to add pressed/active button interaction. 💪
## **_Your hover interaction 👇_**
![image](https://user-images.githubusercontent.com/106119105/213693763-3415c6ab-1145-4530-8b65-9c58909265be.png)

## **_Figma button interaction 👇_**
![image](https://user-images.githubusercontent.com/106119105/213693957-a06cfd62-df56-4799-9d08-05ce80954433.png)

Reply:
- [ ] It is clear that you have given your very best in adding button interactivity. We still have more work to be done here. Please note that on hover, there is a box-shadow and when pressed, there is a color change. Note that there is no scaling effect as you have implemented.  Please find the styling for each state below. You can also find them in the figma design.

**hover**
```
display: flex;
flex-direction: row;
align-items: center;
padding: 12px;
gap: 10px;

position: absolute;
width: 91px;
height: 48px;
left: 331px;
top: calc(50% - 48px/2 - 76.5px);

/* Theme 2 / Primary 2 */

background: #36B37F;
box-shadow: 0px 8px 16px rgba(54, 179, 127, 0.24);
border-radius: 4px;
```
**Pressed**
```
/* Button / Pressed */


/* Auto layout */

display: flex;
flex-direction: row;
align-items: center;
padding: 12px;
gap: 10px;

position: absolute;
width: 91px;
height: 48px;
left: 551px;
top: calc(50% - 48px/2 - 76.5px);

/* Theme 2 / Primary 1 */

background: #008552;
border-radius: 4px;
```